corehq/apps/export/custom_export_helpers.py

```diff
@@ -519,6 +519,7 @@ def get_context(self):
 
 def make_custom_export_helper(request, export_type, domain=None, export_id=None):
     export_type = export_type or request.GET.get('request_type', 'form')
+    # NOTE: minimal is a hack so that some domains can load the exports page properly :-(
```

Comment:
this was because loading the full page was too slow? would be nice to have just a tiny bit more context in this comment